src/translations/screens/subscreens/Information.ts

```diff
@@ -111,7 +111,7 @@ const SelectStartScreenTexts = {
       ),
       part1Bullet3: _(
         `${bulletPoint} at mobiletelefonen fungerer normalt, skjermen er leselig og har nok batterikapasitet så lenge reisen varer slik at gyldig billett kan fremvises. Dersom du ikke kan fremvise billett, vil det anses som å reise uten gyldig billett.`,
-        `${bulletPoint} ensure the mobile phone works properly, the screen is readable and the battery has enough power to present a valid ticket during the whole trip. If you cannot provide a valid ticket upon inspection, you will be considered as travelling without a ticket.`,
+        `${bulletPoint} ensure the phone works properly, the screen is readable and the battery has enough power to present a valid ticket during the whole trip. If you cannot provide a valid ticket upon inspection, you will be considered as travelling without a ticket.`,
```

Comment:
I stedet for "ensure" ville jeg nok heller sagt "Make sure your phone works properly", men hvordan sjekker man egentlig det? Usikker på hva vi skal be brukeren om å gjøre her. @JuliaOppegard @monopoint

Reply:
Enig i Make sure. Ellers tror jeg det vi prøver å kommunisere er at hvis du reiser med en ødelagt telefon kan du risikere at du ikke får vist billett.

Reply:
Det er en litt større context her. dette er et punkt i en liste over ansavarsområder. Helheten er 

As a passenger you are responsiblet to
- ensure the phone works properly (...) 

uansett, jeg endrer til "make sure" og så kan dere opprette en ny sak hvis det trengs en runde til